oozie/src/main/java/org/apache/falcon/logging/JobLogMover.java

```diff
@@ -84,17 +85,23 @@ public void moveLog(WorkflowExecutionContext context){
 
     public int run(WorkflowExecutionContext context) {
         try {
-            OozieClient client = new OozieClient(context.getWorkflowEngineUrl());
+            String engineUrl = context.getWorkflowEngineUrl();
+            if (StringUtils.isBlank(engineUrl)) {
+                LOG.warn("Unable to retrieve workflow url for {} with status {} ",
+                        context.getWorkflowId(), context.getWorkflowStatus());
+                return 0;
```

Comment:
there could be a case where the oozie host has been moved from when the instance ran and the current running oozie.
In which case this will not work. Should we not try with the current oozie url? to get the status of the workflow id?